client/client.go

```diff
@@ -45,17 +58,26 @@ type LambdaClient struct {
 	rules   map[string]bool
 }
 
+func (c *LambdaClient) buildHeaders() map[string]string {
+	policy, _ := json.Marshal(&policy{
+		Rules: c.rules,
+	})
+	return map[string]string{
```

Comment:
Is there any reason why this can't just be `http.Header`? It's just a wrapper around `map[string][]string`. Is this imposed on us by genqlient?

Reply:
`http.Header` is an alias for `map[string][]string` https://pkg.go.dev/net/http#Header, but the invoked lambda functions expects `map[string]string` more or less. (Not sure if this a requirement from lambda, `marketplace-service`, `koa` or something else). I tried using `http.Header` but the request failed with a 500 error